### PR TITLE
[DNM until v2.10.1 is out] [v2.10] Bump ingress-backend

### PR DESCRIPTION
**Pull request by chiukapoor**
### Updates
- Bump ingress-backend to `1.5-rancher2 `

### Issue
- https://github.com/rancher/rancher/issues/48034

### CODE REVIEW

File: pkg/rke/k8s_rke_system_images.go
Comment (jiaqiluo) [inline pkg/rke/k8s_rke_system_images.go:1]: Based on [the comparison](https://github.com/rancher/kontainer-driver-metadata/compare/release-v2.10...dev-v2.10), there will be 3 new versions to be released: v1.29.11-rancher1-1, v1.30.7-rancher1-1, and v1.31.3-rancher1-1. 

it looks like the template for v1.31.3-rancher1-1 is not updated in the PR.

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: pkg/rke/k8s_rke_system_images.go
Comment (chiukapoor) [reply pkg/rke/k8s_rke_system_images.go:1]: Thank you for pointing that out, I have updated the code.

**Review by jiaqiluo — APPROVED** [review]
lgtm

**Review by snasovich — CHANGES_REQUESTED** [review]
Since this is not getting released with v2.10.1, we will need to redo this PR to target versions to be added for v2.10.2 instead.

**Pull request closed without merging**